pom.xml

```diff
@@ -49,6 +49,16 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
+			<version>2.7.5</version>
```

Comment:
Deze <version> tag is niet nodig. Je gebruikt nu een oude versie. We zijn inmiddels als bij 3.x.x, wat beter aansluit bij de Springboot versie die je gebruikt